wiki/Beatmaps/Genre_and_language/en.md

```diff
@@ -32,6 +43,10 @@ The [beatmap listing](https://osu.ppy.sh/beatmapsets) has filters available for
 | Novelty | Comical or unusual. In osu!, Vocaloid tracks are also put under this genre. |
 | Hip Hop | "Stylized rhythmic music that commonly accompanies rapping," sometimes "sampling beats or bass lines from records" ([*Hip hop music* on Wikipedia](https://en.wikipedia.org/wiki/Hip_hop_music)). |
 | Electronic | Composed digitally or with electronic instruments. |
+| Metal | Having a heavy sound, making use of loud and distorted electric guitars and bass, and containing fast or dense drum rhythms. |
+| Classical | Formal, structured around advanced musical theory, and having roots in old western culture or taking inspiration from it. |
+| Folk | Traditional or personal, casual, and reflecting the identity of a culture or group of people. |
```

Comment:
"or" should probably be replaced with "and/or".  And the "Traditional or personal" would look nicer if it was written as "Traditional/personal"

Reply:
the writing style here already has a lot of lists with "or" or "and", maybe somewhere else I'd make the changes but here it seems fine